manifests/init.pp

```diff
@@ -15,9 +14,9 @@
     require => Group['beoper']
   }
 
-  package { $pkgname:
-    ensure  => present,
-    require => User['beuser'],
+  package { 'vrtsralus':
```

Comment:
Changing from the variable to the hardcoded name renders the params.pp useless. Not sure we want that.